seshdash/views.py

```diff
@@ -607,11 +612,25 @@ def silence_alert(request):
 
 def get_latest_bom_data(request):
     latest_bom = BoM_Data_Point.objects.order_by('-time')[0]
-
     latest_bom_data = []
     latest_bom_data.append({"item": "State of Charge", "value":str(latest_bom.soc) + '%' })
     latest_bom_data.append({"item": "Battery Voltage", "value":latest_bom.battery_voltage})
     latest_bom_data.append({"item": "Consumption Data", "value":round(latest_bom.AC_output_absolute, 2)})
     latest_bom_data.append({"item": "Recent Contact", "value": get_timesince(latest_bom.time)})
 
     return HttpResponse(json.dumps(latest_bom_data))
+def search(request):
+    data=[]
+    sites = Sesh_Site.objects.all()
```

Comment:
You don't want to get all the sites. Notice that a user might have permission to view a subset of sites. Make sure you get the users sites